### DISCUSSION TITLE
Cek Outlier

### DISCUSSION THREAD

**Discussion by ageljonathan** [teamalgoritma/community#479] [Q&A] [answered]
Selamat malam
Saya ingin bertanya

Jika kita memiliki dataframe dengan semua kolom bertipe numerik dan ingin melakukan forecasting menggunakan rm
Saya ingin cek apakah ada outlier
Yang saya cek apakah keseluruhan variabel, variabel prediktor saja, atau variabel target saja? Kalau boleh tolong dijelaskan alasannya juga
Dan kalau yang harus dicek outlier terdiri dari beberapa variabel, apakah kita cek dengan cara boxplot() satu2 variabel tersebut?
Bagaimana kalau variabel yg memiliki outlier ternyata lebih dari satu, variabel yg mana yg kita gunakan utk memfilter datanya?

Terima kasih sebelumnya

**Comment by ageljonathan** [comment]
Mohon maaf saya ingin menambahkan
Atau yang perlu di cek outlier adalah variabel prediktor yang memiliki korelasi siginifikan saja?

**Comment by ageljonathan** [comment]
Izin nambah 1 pertanyaan lagi

Saya sedang mengerjakan capstone ML case prediksi beton
Apakah sah jika saya memfilter 100 baris pertama dataset untuk memenuhi uji asumsi Normality of Residuals, Homoscedasticity, Multicollinearity?
Kalau tidak apa cara lain yang saya bisa lakukan?

**Comment by idoali** [comment, marked as answer]
Selamat pagi Pak Agel, mohon izin menjawab pak

Untuk pertanyaan terakhir dulu pak, jawabannya tidak bisa pak. Uji asumsi tersebut harus di uji ke semua data. 

Untuk pertanyaan sebelumnya yaitu terkait dengan outlier. Salah satu alasan mengapa outlier itu dibuang adalah karena outlier bisa mempengaruhi prediksi model nanti. Ditakutkan kalau ada outlier di data training, nanti modelnya akan jelek. Untuk kasus bapak, bisa dicoba-coba pak apakah outlier punya pengaruh atau tidak ke kualitas model.

Ingat pak, tidak perlu membuang outlier jika kita masih bisa mendapatkan model yang bagus.

**Comment by ageljonathan** [comment]
Sip mas Idoo
Thank youu
